ldms/python/ldms.pyx

```diff
@@ -1410,7 +1410,7 @@ cdef class DirSet(object):
         self.data_gn = ds.meta_gn
         self.timestamp = (ds.timestamp.sec, ds.timestamp.usec)
         self.duration = (ds.duration.sec, ds.timestamp.usec)
-        self.info = { ds.info[i].key : ds.info[i].value \
+        self.info = { ds.info[i].key.decode('utf-8') : ds.info[i].value.decode('utf-8') \
```

Comment:
This is a nit, but the decode should probably be xx.decode(encoding='utf-8'). It works like it currently does because encoding happens to be the 1st keyword parameter. If they change that in the future, this could break.

Reply:
Good point, I just force pushed an update.